CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-
+- Updated deprecation warnings for use of nid based calls to capmc
```

Comment:
Thank you for updating the CHANGELOG. I was impressed with how useful it was when I had to tell Dennis what other products were missing out on by not upgrading to the latest versions of the Helm charts and Docker images we provide for import-config, cf-gitea-config, etc. Without that, it would have been a lot of traipsing through git commit history and Jira, so bless Randy for doing a good job on the CHANGELOG.